src/Resque/Component/Job/Model/JobInterface.php

```diff
@@ -67,6 +67,8 @@ public function getArguments();
      */
     public function encode();
 
+    public function getOriginQueue();
```

Comment:
Yep our procline reads {$queue}:{$job} this makes it much easier to see what is going on, expecially with short lived jobs which are too quick to lookup.

I would much rather see deployment:[md5] than [md5] from a ops point of view. Also makes it easier to write a monitoring rule for monitoring the existence of queue processes.